__tests__/app.test.js

```diff
@@ -20,4 +25,14 @@ describe.only("GET /api", () => {
         console.log(endpointsJson)
       });
   });
+  test("200: Responds with an array of object with slug and description keys",()=>{
+    return request (app)
+    .get("/api/topics")
+    .expect(200)
+    .then(({body: {topics}})=>{
+      console.log(topics,"<---- this is topic")
```

Comment:
Make sure to remove any stray console.logs() before committing + pushing. We only use this to debug, so it shouldn't be in the finished end product. By the time we put our code up for code review, we are essentially saying "This piece of code is perfect and can be used as is"! 😄